### PR TITLE
Fix pod lib lint for watchOS + add CI test

### PR DESCRIPTION
**Pull request by ryanwilson**
#no-changelog

### CODE REVIEW

File: FirebaseDatabase/Tests/Helpers/FIRFakeApp.m [hunk 1]
```diff
@@ -47,23 +47,6 @@ @interface FIRComponentContainer (TestInternal)
 @property(nonatomic, strong) NSMutableDictionary<NSString *, FIRComponentCreationBlock> *components;
 @end
 
-@interface FIRComponentContainer (TestInternalImplementations)
```
Comment (ryanwilson) [inline FirebaseDatabase/Tests/Helpers/FIRFakeApp.m:50]: These are already defined in FIRComponentTestUtilities.m in SharedTestUtilities

File: FirebaseDatabase/Sources/Realtime/FWebSocketConnection.m [hunk 2]
```diff
@@ -327,20 +327,22 @@ - (void)URLSession:(NSURLSession *)session
 }
 
 - (void)receiveWebSocketData {
+    __weak FWebSocketConnection *weakSelf = self;
```
Comment (maksymmalyhin) [inline FirebaseDatabase/Sources/Realtime/FWebSocketConnection.m:330]: I find `__weak __autotype weakSelf = self` as it is more compact and is better handled by Xcode refactoring (no need to be changed), but it's definitely up to you.

File: FirebaseDatabase/Sources/Realtime/FWebSocketConnection.m [hunk 3]
```diff
@@ -327,20 +327,22 @@ - (void)URLSession:(NSURLSession *)session
...
     [self.webSocketTask receiveMessageWithCompletionHandler:^(
                             NSURLSessionWebSocketMessage *_Nullable message,
                             NSError *_Nullable error) {
+      FWebSocketConnection *strongSelf = weakSelf;
```
Comment (maksymmalyhin) [inline FirebaseDatabase/Sources/Realtime/FWebSocketConnection.m:334]: nit: I think the common pattern to exit early if `strongSelf == nil`, but it'll be fine if we don't in this particular case.

**Review by maksymmalyhin — APPROVED** [review]

File: FirebaseDatabase/Sources/Realtime/FWebSocketConnection.m [hunk 4]
```diff
@@ -327,20 +327,26 @@ - (void)URLSession:(NSURLSession *)session
 }
 
 - (void)receiveWebSocketData {
+    __weak __autotype *weakSelf = self;
```
Comment (maksymmalyhin) [inline FirebaseDatabase/Sources/Realtime/FWebSocketConnection.m:330]: I think there is a typo here and it should be `__weak __auto_type weakSelf = self;`. I can post a commit if it's already late for you.

**Pull request merged**